tests/test_maya.py

```diff
@@ -160,7 +160,7 @@ def test_slang_date():
 
 def test_slang_time():
     d = maya.when('1 hour ago')
-    assert d.slang_time() == 'an hour ago'
+    assert d.slang_time() == '1 hour ago'
```

Comment:
Let me know your thoughts here. I feel it's fine, but can see why it wouldn't be :cake: